src/main/java/org/alliancegenome/curation_api/jobs/executors/LoadFileExecutor.java

```diff
@@ -190,4 +191,15 @@ public void runCleanup(DiseaseAnnotationService service, BulkLoadFileHistory his
 		ph.finishProcess();
 		
 	}
+	
+	protected void failLoad(BulkLoadFile bulkLoadFile, Exception e) {
+		Set<String> errorMessages = new HashSet<String>();
+		errorMessages.add(e.getMessage());
+		errorMessages.add(e.getLocalizedMessage());
+		if (e.getCause() != null)
+			errorMessages.add(e.getCause().getMessage());
```

Comment:
Not sure if you will need to do the while loop to get the "rootCause" but we can do with this.

Reply:
PS... good idea with the set.

Reply:
Added the while loop and also changed the Set type to LinkedHashSet in order to preserve order of insertion

Reply:
Nice.